package.json

```diff
@@ -114,7 +114,8 @@
     "rollup-plugin-ts": "^2.0.5",
     "semantic-release": "^19.0.2",
     "typescript": "^4.3.5",
-    "vite": "^2.9.1"
+    "vite": "^2.9.1",
+    "vuepress": "^1.9.7"
```

Comment:
by adding vuepress as a root devDep, we should be able to run those npm scripts without issue now. Added benefit that we really only have 1 package.json to manage.

The other package.json is just sort of a hack to tell npm to ignore the fact the root package.json is a `module` so it stops asking us to use `.mjs` inside the vuepress dirs.